src/redux.js

```diff
@@ -51,6 +60,21 @@ export default function rootReducer(state = initialState, action) {
       data: action.data
     };
   }
+  if (action.type === SET_METADATA_BY_COLUMN) {
+    const prevMetaData = state.metaDataByColumn[action.column]
```

Comment:
What is `prevMetaData` intended to be used for?

Reply:
nothing! It was leftover from a different path I tried. Deleted! Thanks for catching.